firefox-ios/nimbus.fml.yaml

```diff
@@ -24,6 +24,7 @@ include:
   - nimbus-features/homescreenFeature.yaml
   - nimbus-features/messagingFeature.yaml
   - nimbus-features/onboardingFrameworkFeature.yaml
+  - nimbus-features/preferSwitchToOpenTabOverDuplicateFeature.yaml
```

Comment:
Sure thing, what do you think @adudenamedruby?

Reply:
Yeah,, I'd personally put it in there, just to avoid bloat on "features" and because I think it kinda makes sense for it to be there. But I'm 60/40 on it, so I'm good either way, really.